packages/falso/package.json

```diff
@@ -34,5 +34,8 @@
   "devDependencies": {
     "@types/seedrandom": "3.0.1",
     "@types/uuid": "8.3.4"
+  },
+  "exports": {
+    "./i18n/*": "./i18n/*/index.cjs.js"
```

Comment:
Can you verify it works with web bundlers such as Wepback, please? I' suggesting to build the library and `npm link` to `dist`

Reply:
Checked for webpack, it gave error for missing entry of `.` in exports, so added that and it worked fine.

Reply:
I'm not sure about this approach because you're referring only to cjs modules here, and web bundles can't tree-shake it.

Reply:
Other approach could be to have package.json for each language, and it contains `main` and `module` entries in it and remove `exports` from main package.json. Should I check that? We will need to take care that users still need to only install main package only.

Reply:
> We will need to ensure that users still need to install the main package only.

That's not an issue because it's tree-shakable. It'll not be bundled if they do not use the i18n functions. 


There's a problem we did not think of - name collision. We should prefix the i18n function with the locale (e.g. `ruRandFullName)`

Let's try the other builder with the entry point support regarding the build.

Reply:
With the prefix in place we can just export the i18n functions in the regular index.ts file, right? 🤔 No additional entry point is needed.

Reply:
@michaelxvoelker feel free to clone the repo or even the branch I created and try out what you mentioned. Let me know if it works.

Reply:
Tried node builder, but doesn't produce results the way we want. I will research more.